main/templates/main/index.html

```diff
@@ -5,8 +5,14 @@
 
 {% block content %}
 <div class="col">
-  <a href="{% url 'main:boot_process' %}" class="btn btn-primary">Boot</a>
-  {% render_table table %}
+  <form method="post", action="{% url 'main:process_action' %}">
+    {% csrf_token %}
+    <a href="{% url 'main:boot_process' %}" class="btn btn-primary">Boot</a>
+    <input type="submit" value="Restart" class="btn btn-success", name="action", onclick="return confirm('Restart selected processes?')">
```

Comment:
You can use the [bootstrap_button](https://django-bootstrap5.readthedocs.io/en/latest/templatetags.html#bootstrap-button) template tag to simplify these.

Reply:
I'm not entirely sure I understand the benefit of this. What is simplified? I still need to add all of those fields, right?

Reply:
Well primarily it handles the styling.

Reply:
Not that I don't want to change this, but
1. I'm not sure how to use the tag, and
2. It seems to me that what I use is perfectly fine with bootstrap style, [according to the docs](https://getbootstrap.com/docs/5.0/components/buttons/#button-tags).

Reply:
Well lets leave it for now. See https://github.com/ImperialCollegeLondon/dune_processes/blob/4b7496bc957a32155181e66c4e9277bf25450157/main/templates/registration/login.html#L24 for reference.

Reply:
Yeah, I know how to use the tag in simple cases, but in here I had way more arguments that I don't know how to include in case of the tag.

Reply:
Ah, you just pass them as extra keyword arguments to the tag e.g.:
`{% bootstrap_button button_type="submit" content="Login" onclick="return confirm('Restart selected processes?')" %} `